semcore/utils/theme/light.json

```
@@ -2341,6 +2425,54 @@
         "type": "typography"
       }
     },
+    "header": {
+      "bg": {
+        "value": "{violet.dusty.700}",
+        "type": "color"
+      },
+      "border": {
+        "primary": {
+          "value": "{violet.dusty.700}",
+          "type": "color"
+        },
+        "secondary": {
+          "value": "rgba(255,255,255,0.15)",
+          "type": "color"
+        }
+      }
+    },
+    "left-nav": {
```

```suggestion
    "sidebar-nav": {
```
I think we can avoid left/right here

---

Thank you! Yes, I guess this could be a good choice.